.vscode/cspell.json

```diff
@@ -141,6 +144,8 @@
     "LPBYTE",
     "LPSTR",
     "LPWSTR",
+    "MAKELANGID",
+    "mbedtls",
     "mbps",
```

Comment:
When do we decide when one spelling exception belongs in this global cspell.json list vs a local exception within the file via a comment?

My heuristic would be if I know something is spelt incorrectly (intentionally) like "abcdabcd" in a test file, then I would create a local exception just for that file. If, however, it is spelt correctly but there's a limitation of the spell checker and could apply to more files, then I put it in the global `cspell.json`. Is that similar to how you think about it?

Reply:
So far, I am looking whether the word is being used in 1-2 files vs 10s of files.
These specifically, `MAKELANGID` is a macro from `winnt.h` (https://learn.microsoft.com/en-us/windows/win32/api/winnt/nf-winnt-makelangid) - so kind of makes sense to add it to global, and `mbedtls` (all lowercase) is a verbatim library name - https://github.com/Mbed-TLS/mbedtls